spyder/plugins/console/plugin.py

```diff
@@ -131,16 +136,39 @@ def get_sys_path(self):
         """
         return self.get_widget().get_sys_path()
 
-    def handle_exception(self, text, is_traceback, is_pyls_error=False,
-                         is_faulthandler_report=False):
+    @Slot(dict)
+    def handle_exception(self, error_data):
         """
         Handle any exception that occurs during Spyder usage.
+
+        Parameters
+        ----------
+        error_data: dict
+            The dictionary containing error data. The expected keys are:
+            >>> error_data= {
+                "text": str,
+                "is_traceback": bool,
+                "repo": str,
+                "title": str,
+                "label": str,
+                "steps": str,
+            }
+
+        Notes
+        -----
+        The `is_traceback` indicates if `text` contains, plain text or a
+        python error traceback.

```

Comment:
Please change this to

    The `is_traceback` key indicates if `text` contains plain text or a
    Python error traceback.

Reply:
Fixed